client/keys.go

```diff
@@ -14,7 +14,7 @@
 
 package client
 
-//go:generate codecgen -r "Node|Response|Nodes" -o keys.generated.go keys.go
+//go:generate codecgen -d 1819 -r "Node|Response|Nodes" -o keys.generated.go keys.go
```

Comment:
why do we need this? is the number supposed to be random when each time the code is generated?